www/js/bluetooth/BluetoothScanPage.tsx

```diff
@@ -90,10 +92,22 @@ const BluetoothScanPage = ({ ...props }: any) => {
       ...prevDevices,
       [uuid]: {
         ...prevDevices[uuid],
-        monitorResult: result,
+        monitorResult: status ? result : undefined,
+        rangeResult: undefined,
         in_range: status,
       },
     }));
+    let { monitorResult: _, in_range: _, ...noResultDevice } = sampleBLEDevices[uuid];
```

Comment:
Bundling JS fails at this line.
What is it supposed to do anyway? It doesn't appear to do anything

Reply:
It removes `monitorResult` and `in_range` from `sampleBLEDevices[uuid]` and retains the rest in `noResultDevice`
https://stackoverflow.com/a/34710102

what is the error that you get? Is this primarily in production mode? I am able to build in dev mode just fine.

Reply:
I dont' see anywhere that `noResultDevice` is used, though

```SyntaxError: ... Identifier '_' has already been declared.```
it's trying to destructure the object and assign both `monitorResult` and `in_range` to a variable called `_`.

Yes this happened with `build-prod-android`

Reply:
> Yes this happened with build-prod-android

Ah I haven't tried the prod builds yet. A simple workaround would be to use `{ monitorResult: p0, in_range: p1, ...}` to avoid reusing the `_` variable.

> I dont' see anywhere that noResultDevice is used, though

Ah, before my most recent set of commits, when i used a newly added native mock function, I was saving the values directly using the usercache. See https://github.com/e-mission/e-mission-phone/pull/1144/commits/7cb388227e52369725ab4c20c5692d433dbc6417

When I changed to use mocks, I forgot to clean these up. I can do that now, or you can comment them out to unblock yourself.